sdk/go/common/workspace/creds.go

```diff
@@ -251,3 +251,123 @@ func StoreCredentials(creds Credentials) error {
 
 	return nil
 }
+
```

Comment:
Can we add test coverage for the functionality here?

Reply:
I will open an issue to add tests here - we may need to think about how to do that better as this writes a file to disk